src/applications/discharge-wizard/constants/index.js

```diff
@@ -1,4 +1,4 @@
-import * as options from 'platform/static-data/options-for-select';
+export const DW_UPDATE_FIELD = 'DW_UPDATE_FIELD';
```

Comment:
Can you scope this constant a bit more? Maybe something like:
```js
export const UPDATE_FIELD = 'discharge-wizard/UPDATE_FIELD'
```

That way if another app on VA.gov in the future also has an action that dispatches the constant `'UPDATE_FIELD'` they won't accidentally fire off the wrong action type 🙂

Reply:
Oh nice catch, I can add it.